packages/core/expressions/src/components/ExpressionsEditor.vue

```diff
@@ -18,18 +18,26 @@ import { registerLanguage, registerTheme, theme } from '../monaco'
 let editor: Monaco.editor.IStandaloneCodeEditor | undefined
 let editorModel: Monaco.editor.ITextModel
 
-const parse = (expression:string, schema: AtcSchema) => Parser.parse(expression, schema)
-
 const { debounce } = useDebounce()
 
 const props = withDefaults(defineProps<{
   schema: Schema,
   parseDebounce?: number,
   inactiveUntilFocused?: boolean,
+  allowEmptyInput?: boolean,
+  editorOptions?: Monaco.editor.IEditorOptions
 }>(), {
   parseDebounce: 500,
```

Comment:
nit: Would we need to specify a default value for the new prop?

<img width="790" alt="Screenshot 2024-11-19 at 11 46 53" src="https://github.com/user-attachments/assets/6799e0b3-a687-439c-a6f9-c8e7d949e71a">


```suggestion
  parseDebounce: 500,
  editorOptions: undefined,
```